QGCCommon.pri

```diff
@@ -194,14 +198,19 @@ WindowsBuild {
 
 ReleaseBuild {
     DEFINES += QT_NO_DEBUG
-    WindowsBuild {
-        # Use link time code generation for better optimization (I believe this is supported in MSVC Express, but not 100% sure)
-        QMAKE_LFLAGS_LTCG = /LTCG
-        QMAKE_CFLAGS_LTCG = -GL
+    CONFIG += force_debug_info  # Enable debugging symbols on release builds
+    CONFIG += ltcg              # Turn on link time code generation
```

Comment:
The error suggests this fix. You can apply or I will when I get out of this meeting.